path_config.go

```diff
@@ -119,6 +119,11 @@ func (b *kubeAuthBackend) pathConfigRead(ctx context.Context, req *logical.Reque
 	} else if config == nil {
 		return nil, nil
 	} else {
+		var tokenReviewerJWTSet bool
+		if config.TokenReviewerJWT != "" {
+			tokenReviewerJWTSet = true
+		}
+
```

Comment:
We can replace the variable assignment here with an inline boolean per the suggestion for line 136
```suggestion
```

Reply:
Thank you!